OrbitCore/CallstackData.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "CallstackData.h"
+
+#include "Callstack.h"
+#include "Capture.h"
+#include "SamplingProfiler.h"
+
+using orbit_client_protos::CallstackEvent;
+
+void CallstackData::AddCallstackEvent(CallstackEvent&& callstack_event) {
+  CallstackID hash = callstack_event.callstack_hash();
+  CHECK(HasCallStack(hash));
+  callstack_events_.push_back(std::move(callstack_event));
+}
+
+void CallstackData::AddUniqueCallStack(CallStack call_stack) {
+  absl::MutexLock lock(&unique_callstacks_mutex_);
+  CallstackID hash = call_stack.GetHash();
+  unique_callstacks_[hash] = std::make_shared<CallStack>(std::move(call_stack));
+}
+
+void CallstackData::AddCallStackFromCallstackData(
```

Comment:
I guess this is used for selections, where other is the global report? Maybe make this a bit more clear in the naming.
E.g. `other` -> `superset`/`all_callstacks`/`known_callstacks`.

Reply:
Renamed the argument and the function.